### PR TITLE
Fixes #2400 : Add a "Send updated nutrition photo" button below nutrition image

### LINKED ISSUE #2400: Add a "Send update nutrition photo" button below nutrition image
Add a "Send update nutrition photo" button below nutrition image

### PR DESCRIPTION
**Pull request by mustaqmustu**
## Related issues and discussion
#2400 

Screenshots:

 
 ## Checklist
 
Make sure you've done all the following (You can delete the checklist before submitting)
 
 - [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.
 - [x] Code is well documented
 - [x] Include unit tests for new functionality
 - [x] All user-visible strings are made translatable
 - [x] Code passes Travis builds in your branch
 - [x] If you have multiple commits please combine them into one commit by squashing them.
 - [x] Read and understood the contribution guidelines .

### CODE REVIEW

**Comment by teolemon** [comment]
How does it behave on new products that don't have a nutrition photo yet ?
We have a logic error on ingredients in https://github.com/openfoodfacts/openfoodfacts-androidapp/issues/2316

**Comment by mustaqmustu** [comment]
> How does it behave on new products that don't have a nutrition photo yet ?
> We have a logic error on ingredients where we show Send Updated Photo while we have no initial photo

When there is no photo of nutrition..its behaving like this:
ezgif com-video-to-gif(6)

**Comment by mustaqmustu** [comment]
> How does it behave on new products that don't have a nutrition photo yet ?
> We have a logic error on ingredients where we show Send Updated Photo while we have no initial photo
When you dont have a nutrition photo,it behaves like this:
ezgif com-video-to-gif(7)

**Comment by mustaqmustu** [comment]
It works like these when the product doesnt have a nutrition photo yet
[20190318_184206.zip](https://github.com/openfoodfacts/openfoodfacts-androidapp/files/2978249/20190318_184206.zip)

**Comment by teolemon** [comment]
Ok, it doesn't make send to ask for an updated picture if we don't have an initial one. Can you add a condition on the ingredient and nutrition "Take an updated picture" to check whether there's an nutrition image (for the nutrition button) or an ingredient image (for the ingredient button) for the product ?

**Comment by mustaqmustu** [comment]
> Ok, it doesn't make send to ask for an updated picture if we don't have an initial one. Can you add a condition on the ingredient and nutrition "Take an updated picture" to check whether there's an nutrition image (for the nutrition button) or an ingredient image (for the ingredient button) for the product ?

@tolemon sure :)

**Comment by mustaqmustu** [comment]
> Ok, it doesn't make send to ask for an updated picture if we don't have an initial one. Can you add a condition on the ingredient and nutrition "Take an updated picture" to check whether there's an nutrition image (for the nutrition button) or an ingredient image (for the ingredient button) for the product ?

@teolemon ,I have a doubt...so according to you,send an update image button should appear only when that product has already an image in nutrition and ingredients right?

**Comment by mustaqmustu** [comment]
> Ok, it doesn't make send to ask for an updated picture if we don't have an initial one. Can you add a condition on the ingredient and nutrition "Take an updated picture" to check whether there's an nutrition image (for the nutrition button) or an ingredient image (for the ingredient button) for the product ?
Is this ok?
ezgif com-video-to-gif(9)
If it is okay,I will made changes in code according to it.

**Comment by mustaqmustu** [comment]
@teolemon ,I have made the required changes

File: app/src/main/res/layout/fragment_nutrition_product.xml [hunk 1]
```diff
@@ -36,7 +36,15 @@
                 android:scaleType="fitCenter"
                 app:srcCompat="@drawable/ic_add_a_photo_black_48dp" />
 
-
+            <Button
+                android:drawableLeft="@drawable/ic_add_a_photo_black_18dp"
```
Comment (brandenfung) [inline app/src/main/res/layout/fragment_nutrition_product.xml]: Could you use the `BorderButton` style to ensure that all buttons have the same design? For example, see button style (change_ing_img) in fragment_ingredients_product.xml:

image

Comment (brandenfung) [inline app/src/main/res/layout/fragment_nutrition_product.xml]: Could you use the `BorderButton` style to ensure the design of all buttons are the same? For example take a look at the button (with id change_ing_img) in `fragment_ingredients_product.xml` :
image

Comment (mustaqmustu) [reply app/src/main/res/layout/fragment_nutrition_product.xml]: Changes made :)

**Comment by teolemon** [comment]
Merging.

**Pull request merged**